src/components/ContentNode.vue

```diff
@@ -24,6 +25,7 @@ import StrikeThrough from './ContentNode/StrikeThrough.vue';
 const BlockType = {
   aside: 'aside',
   codeListing: 'codeListing',
+  sectionTitle: 'sectionTitle',
```

Comment:
I don't think this line should needed since these enum values refer to content node block types from the Render Node spec (and this is not one).